cronjobs/bsz_util.py

```diff
@@ -0,0 +1,150 @@
+# Python 3 module
+# -*- coding: utf-8 -*-
+
+from ftplib import FTP
+import datetime
+import os
+import re
+import sys
+import time
+import traceback
+import util
+import shutil
+import string
+import tempfile
+
+
+def FoundNewBSZDataFile(link_filename):
+    try:
+        statinfo = os.stat(link_filename)
+        file_creation_time = statinfo.st_ctime
+    except OSError as e:
+        util.Error("in FoundNewBSZDataFile: Symlink \"" + link_filename + "\" is missing or dangling!")
```

Comment:
I think util.Error retrieves the name of the calling function from the stack => you can leave out "n FoundNewBSZDataFile: ".